starter-code/routes/signup.js

```diff
@@ -0,0 +1,63 @@
+const express = require("express");
+const userModel = require('../model/user');
+const bcrypt = require("bcrypt");
+const router = express.Router();
+const bcryptSalt = 10;
+
+router.get('/signup',(req, res, next) => {
+  if (req.session.currentUser) {
```

Comment:
again, instead of writing this guard two times (in each of the routes in this router module), you could have added a middleware at the top - see comment above in `routes/signin`